lib/config.coffee

```diff
@@ -45,26 +45,32 @@ module.exports = {
     type: 'boolean'
     default: true
     order: 10
+  removePackage:
+    title: 'Remove unused packages'
+    description: 'Packages installed but not in the backup will be removed when restoring backups'
+    type: 'boolean'
+    default: false
+    order: 11
```

Comment:
Is there a reason `removePackage` needs to be inserted here, thus pushing the `order:` attribute of everything that follows?

Could it have been appended to the end of the list instead?